lib/elixir/lib/float.ex

```diff
@@ -30,6 +32,12 @@ defmodule Float do
 
   There are also other known problems like flooring or rounding numbers. See
   `round/2` and `floor/2` for more details about them.
+
+  To learn more about floting-point arithmetic visit:
+
+    * http://0.30000000000000004.com/
```

Comment:
Will this be a clickable link in the docs?

Reply:
I checked and there's no autolinking in ex_doc. I'll correct it